Makefile

```
@@ -702,13 +702,16 @@ lint-go:
 	golangci-lint run -c .golangci.yml --build-tags='$(LIBFIDO2_TEST_TAG) $(TOUCHID_TAG) $(PIV_TEST_TAG)' $(GO_LINT_FLAGS)
 
 .PHONY: fix-imports
-fix-imports: GOLANG_LINT_ARGS = --build-tags='$(LIBFIDO2_TEST_TAG) $(TOUCHID_TAG) $(PIV_TEST_TAG)' --fix
 fix-imports:
-	golangci-lint run -c .golangci.yml $(GOLANG_LINT_ARGS)
-	cd api/ && golangci-lint run -c ../.golangci.yml $(GOLANG_LINT_ARGS)
-	cd build.assets/tooling && golangci-lint run -c ../../.golangci.yml $(GOLANG_LINT_ARGS)
-	cd .cloudbuild/scripts && golangci-lint run -c ../../.golangci.yml $(GOLANG_LINT_ARGS)
-	cd assets/backport && golangci-lint run -c ../../.golangci.yml $(GOLANG_LINT_ARGS)
+	make -C build.assets/ fix-imports
+
+.PHONY: fix-imports/host
+fix-imports/host:
+	@if ! type gci 2>&1 >/dev/null; then\
```

Redirection order matters:

```suggestion
	@if ! type gci >/dev/null 2>&1; then\
```

---

> Redirection order matters
Aware, but the fight with the "dot" got me distracted, and I forgot to fix that. Thanks